<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_source</class>
 <widget class="QDialog" name="Add_source">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Название источника</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myres.qrc">
         <normaloff>:/images/keyboard_icon-icons.com_71162.png</normaloff>:/images/keyboard_icon-icons.com_71162.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myres.qrc">
         <normaloff>:/images/keyboard_icon-icons.com_71162.png</normaloff>:/images/keyboard_icon-icons.com_71162.png</iconset>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Мощность при последней поверке</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myres.qrc">
         <normaloff>:/images/keyboard_icon-icons.com_71162.png</normaloff>:/images/keyboard_icon-icons.com_71162.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myres.qrc">
         <normaloff>:/images/keyboard_icon-icons.com_71162.png</normaloff>:/images/keyboard_icon-icons.com_71162.png</iconset>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Текущая активность</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Активность при
изготовлении (МБк)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myres.qrc">
         <normaloff>:/images/keyboard_icon-icons.com_71162.png</normaloff>:/images/keyboard_icon-icons.com_71162.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Дата изготовления</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="7" column="4">
      <widget class="QComboBox" name="comboBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Изотоп</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Текущая мощность дозы</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Период полураспада, лет</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myres.qrc">
         <normaloff>:/images/keyboard_icon-icons.com_71162.png</normaloff>:/images/keyboard_icon-icons.com_71162.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Дата проведения поверки</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>МБк</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myres.qrc">
         <normaloff>:/images/keyboard_icon-icons.com_71162.png</normaloff>:/images/keyboard_icon-icons.com_71162.png</iconset>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="myres.qrc">
         <normaloff>:/images/keyboard_icon-icons.com_71162.png</normaloff>:/images/keyboard_icon-icons.com_71162.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Заводской номер</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Дата последней поверки</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Текущая дата</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_9">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myres.qrc"/>
 </resources>
 <connections/>
</ui>
